src/components/controllers/repl/runner.js

```diff
@@ -99,7 +104,7 @@ export default class Runner extends Component {
 		}
 		if (vnode) {
 			try {
-				this.root = render(vnode, this.base, this.root);
+				this.root = render(vnode, this.base);
```

Comment:
I wonder if we should wrap a component with a `componentDidCatch` lifecycle all around the users code.

Reply:
ooh I like that idea!